airflow/providers/google/common/hooks/base_google.py

```diff
@@ -642,8 +642,10 @@ def __init__(
         *,
         project: str | None = None,
         session: ClientSession | None = None,
+        scopes: Sequence[str] | None = None,
     ) -> None:
-        super().__init__(session=cast(Session, session))
+        _scopes: list[str] | None = list(scopes) or None
```

Comment:
```suggestion
        _scopes: list[str] | None = list(scopes) if scopes else None
```

Reply:
exactly, list(None) will raise an exception

Reply:
Thanks for fixing that! I was intended to do it this morning but you guys were faster!